libraries/script-engine/src/ScriptEngine.cpp

```diff
@@ -2363,30 +2361,57 @@ void ScriptEngine::entityScriptContentAvailable(const EntityItemID& entityID, co
         }
     }
     else {
```

Comment:
Someone messed up the else here can you fix it back to `} else {` while you're touching this file anyway?